service-fabric-cluster-5-node-1-nodetype/azuredeploy.json

```diff
@@ -0,0 +1,592 @@
+      }
+    },
+    "loadBalancedAppPort2": {
+      "type": "int",
+      "defaultValue": 8081,
+      "metadata": {
+        "description": "Input end point2 for the application to use"
+      }
+    },
+    "storageAccountType": {
+      "type": "string",
+      "allowedValues": [
+        "Standard_LRS",
+        "Standard_GRS"
+      ],
+      "defaultValue": "Standard_LRS",
+      "metadata": {
+        "description": "Storage account Type for the xstore used by the VM"
+      }
+    },
+    "diagnosticStorageAccountType": {
+      "type": "string",
+      "allowedValues": [
+        "Standard_LRS",
+        "Standard_GRS"
+      ],
+      "defaultValue": "Standard_LRS",
+      "metadata": {
+        "description": "Storage account Type for the xstore used for stroring diagontics information from the cluster"
+      }
+    }
+  },
+    "variables": {
+      "computeLocation": "[parameters('clusterLocation')]",
+      "dnsName": "[parameters('clusterName')]",
+      "vmStorageAccountName": "[concat(parameters('clusterName'),'1')]",
+      "vmName": "VM",
+      "applicationStartPort": "20000",
+      "applicationEndPort": "30000",
+      "ephemeralStartPort": "49152",
+      "ephemeralEndPort": "65534",
+      "publicIPAddressName": "PublicIP-VM",
+      "publicIPAddressType": "Dynamic",
+      "vmStorageAccountContainerName": "vhds",
+      "virtualNetworkName": "VNet",
+      "addressPrefix": "10.0.0.0/16",
+      "subnet1Name": "Subnet-1",
+      "subnet2Name": "Subnet-2",
+      "subnet1Prefix": "10.0.0.0/24",
+      "subnet2Prefix": "10.0.1.0/24",
+      "nicName": "NIC",
+      "lbName": "LoadBalancer",
+      "lbIPName": "PublicIP-LB-FE",
+      "availSetName": "AvailabilitySet",
+      "maxPercentUpgradeDomainDeltaUnhealthyNodes": "100",
+      "vmImagePublisher": "MicrosoftWindowsServer",
+      "vmImageOffer": "WindowsServer",
+      "vmImageSku": "2012-R2-Datacenter",
+      "vmImageVersion": "latest",
+      "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+      "subnet1Ref": "[concat(variables('vnetID'),'/subnets/',variables('subnet1Name'))]",
+      "diagnosticStorageAccountName": "[concat(parameters('clusterName'),'2')]",
+      "lbID0": "[resourceId('Microsoft.Network/loadBalancers',concat(variables('lbName'),'-','0'))]",
+      "lbIPConfig0": "[concat(variables('lbID0'),'/frontendIPConfigurations/LoadBalancerIPConfig')]",
+      "lbPoolID0": "[concat(variables('lbID0'),'/backendAddressPools/LoadBalancerBEAddressPool')]",
+      "lbProbeID0": "[concat(variables('lbID0'),'/probes/FabricGatewayProbe')]",
+      "lbHttpProbeID0": "[concat(variables('lbID0'),'/probes/FabricHttpGatewayProbe')]",
+      "vmNodeType0Name": "[concat('NT1', variables('vmName'))]",
+      "vmNodeType0Size": "Standard_A2"
+    },
+    "resources": [
+      {
+        "apiVersion": "2015-05-01-preview",
```

Comment:
Consider making the api version a variable so it's easy to move to newer api versions across all resources below. Also try using the newer 2015-06-15 API